Lib/test/test_functools.py

```diff
@@ -2449,6 +2449,18 @@ def f(*args):
         with self.assertRaisesRegex(TypeError, msg):
             f()
 
+    def test_invalid_positional_argument_singlemethoddispatch(self):
```

Comment:
I would suggest changing the test method name to match the method being tested

```suggestion
    def test_invalid_positional_argument_singledispatchmethod(self):
```

Reply:
Thank you for catching this! And don't worry, everyone's reviews matter :)